torch-compiler/src/test/java/org/brightify/torch/SimpleEntityTest.java

```diff
@@ -65,5 +65,49 @@ public void testEntityWithTwoIdProperties() {
                 .withErrorContaining("There has to be exactly one @Id property in each entity!")
                 .in(JavaFileObjects.forResource("negative/EntityWithTwoIdProperties.java")).onLine(13);
     }
+    
+    @Test
+    public void testEntityWithIntId(){
+        ASSERT.about(javaSource()).that(JavaFileObjects.forResource("negative/EntityWithIntId.java")).
+                processedWith(new TorchCompilerEntrypoint()).failsToCompile().
+                withErrorContaining("@Id has to be type Long.").
+                in(JavaFileObjects.forResource("negative/EntityWithIntId.java")).onLine(9);
+    }
+    
+    @Test
+    public void testEntityWithPrimitiveIntId(){
+        ASSERT.about(javaSource()).that(JavaFileObjects.forResource("negative/EntityWithPrimitiveIntId.java")).
+                processedWith(new TorchCompilerEntrypoint()).failsToCompile().
+                withErrorContaining("@ID has to be type Long.").
+                in(JavaFileObjects.forResource("negative/EntityWithPrimitiveIntId.java")).onLine(9);
+    }
+
+    @Test
+    public void testEntityWithoutIdAnnotation(){
```

Comment:
Right, there was but it is testing slightly different scenario. We should improve the name then.